README.md

```diff
@@ -2,9 +2,9 @@
 
 Telepresence combined with GitHub Actions allows you to run integration tests in your CI pipeline without having to run any dependant service. By connecting to the target Kubernetes cluster, intercepting traffic to the remote service and sending it to an instance of the service running in CI you will be able to test bugfixes, updates and features very easily.
 
-## V1.0
+## v1.0.0
 
-The V1.0 of github actions for Telepresence contains individual actions to:
+The v1.0.0 of GitHub actions for Telepresence contains individual actions to:
```

Comment:
Should be "Telepresence GitHub Actions" instead of "GitHub actions for Telepresence".